src/shared/Login.jsx

```diff
@@ -1,8 +1,28 @@
-import { Link } from "react-router-dom";
+import { GoogleAuthProvider, getAuth, signInWithPopup } from "firebase/auth";
 
 import imgUrl from "../assets/sticky-seacher-logo.png";
+import { app } from "../firebase/firebase";
 
 export default function Login() {
+  const auth = getAuth(app);
+  const authData = async () => {
+    const provider = new GoogleAuthProvider();
+
+    const data = await signInWithPopup(auth, provider);
+
+    const user = data.user;
+    const email = user.email;
+    const accessToken = user.accessToken;
+
+    localStorage.setItem("userEmail", email);
+    localStorage.setItem("userAccessToken", accessToken);
+
+    // eslint-disable-next-line no-unused-vars
+    const userEmail = window.localStorage.getItem("userEmail");
+    // eslint-disable-next-line no-unused-vars
+    const userAccessToken = window.localStorage.getItem("userAccessToken");
```

Comment:
P2: 변수를 만들지 않으면 eslint가 push를 방해하지 않는데, 주석보다는 이런건 어떨까요?

```suggestion
    window.localStorage.getItem("userEmail");
    window.localStorage.getItem("userAccessToken");
```